cmake/Options.cmake

```diff
@@ -127,11 +127,16 @@ endif()
 #
 # e.g. `-DCMAKE_BUILD_TYPE=TESTING`
 ####
-SET(CMAKE_CXX_FLAGS_RELEASE "-std=c++03" CACHE STRING "C++ flags." FORCE)
-SET(CMAKE_C_FLAGS_RELEASE "-std=c99" CACHE STRING "C flags." FORCE)
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
What about setting the standard to C++03 here, then, if the build type is testing set the standard to C++11 within the same conditional as the additional C++ flags?